mifosng-android/src/main/java/com/mifos/mifosxdroid/passcode/PassCodeActivity.java

```diff
@@ -97,12 +98,14 @@ public void passCodeEntered(String passcode) {
         if (isPassCodeLengthCorrect()) {
             if (isPassCodeCorrect) {
                 startDashBoardActivity();
+                finish();
             } else {
                 counter++;
                 passCodeView.clearPasscodeField();
                 Toaster.show(clRootview, R.string.incorrect_passcode);
             }
         }
+
```

Comment:
You haven't added the screenshot/GIF of the changes yet... Add it so that the reviewers (or other org members) don't have to build the app from your repo to observe the changes.

Reply:
Done. I am sorry for carelessness.I will be cautious from next time.